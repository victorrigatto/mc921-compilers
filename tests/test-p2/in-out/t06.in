int var = (1 < 4) && (5 >= 10) || (y != x);
