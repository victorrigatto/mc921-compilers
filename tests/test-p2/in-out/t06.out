Program:
    GlobalDecl:
        Decl: ID(name=var)
            VarDecl:
                Type: int @ 1:1
            BinaryOp: || @ 1:12
                BinaryOp: && @ 1:12
                    BinaryOp: < @ 1:12
                        Constant: int, 1 @ 1:12
                        Constant: int, 4 @ 1:16
                    BinaryOp: >= @ 1:23
                        Constant: int, 5 @ 1:23
                        Constant: int, 10 @ 1:28
                BinaryOp: != @ 1:36
                    ID: y @ 1:36
                    ID: x @ 1:41
